### DISCUSSION TITLE
How to Manage All Merchants, Set Default Settings, and Gather Analytics as a Payment Service Provider?

### DISCUSSION THREAD

**Discussion by mlodhia10** [juspay/hyperswitch#5392] [Q&A]
Hello HyperSwitch Community,

I need some guidance on the best practices and workflow to achieve the following:

Manage All Merchants: What is the recommended approach to efficiently manage all merchants under a single platform? Are there any specific features or modules in HyperSwitch that facilitate this?

Set Default Settings: How can I set default settings (e.g., transaction limits, default payment gateways, etc.) that apply to all merchants? Is there a centralized way to update these settings and propagate them to all merchants?

Gather Analytics: What tools or features within HyperSwitch can be used to gather comprehensive analytics for all merchants? I need to track various metrics such as transaction volumes, success rates, and revenue for each merchant as well as aggregate data for the entire platform.

I would appreciate any insights, best practices, or examples from your experience. Detailed documentation, code snippets, or references to relevant resources would also be highly helpful.

Thank you in advance for your assistance!

**Comment by EbrahimSolomon** [comment]
Use HyperSwitch’s Organization → Merchant → Profile model. Make your platform a Platform Organization so you can programmatically create merchants, generate their API keys, and push a baseline config (connectors, routing, limits). Use Control Centre for centralized routing defaults, and the built-in Analytics plus observability (Grafana/Loki/OpenTelemetry) for per-merchant and aggregate metrics.
